src/Amp/Command/ConfigCommand.php

```diff
@@ -45,20 +45,24 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     );
 
     $output->writeln("");
-    $output->writeln("<info>=============================[ Configure MySQL ]=============================</info>");
+    $output->writeln("<info>=============================[ Configure Database ]=============================</info>");
     $output->writeln("");
     $output->writeln("<info>"
-        ."Amp creates a unique MySQL user for each generated instance.\n"
+        ."Amp creates a unique database user for each generated instance.\n"
         ."To accomplish this amp needs GRANT-level privileges. It is\n"
         ."recommended that you supply the root/administrator credentials\n"
         ."for this task. If you wish to create a new user for amp to use\n"
         ."please assign it appropriate privileges eg:\n\n"
-        ."<fg=cyan;bg=black;option=bold>GRANT ALL ON *.* to '#user'@'localhost' IDENTIFIED BY '#pass' WITH GRANT OPTION</fg=cyan;bg=black;option=bold>"
+// FIXME
+        ."MySQL: <fg=cyan;bg=black;option=bold>GRANT ALL ON *.* to '#user'@'localhost' IDENTIFIED BY '#pass' WITH GRANT OPTION</fg=cyan;bg=black;option=bold>\n"
+        ."PgSQL: <fg=cyan;bg=black;option=bold>\$ createuser --superuser --createdb --createrole -P #user</fg=cyan;bg=black;option=bold>\n"
+        ."       <fg=cyan;bg=black;option=bold>Add 'local all #user md5' to pg_hba.conf</fg=cyan;bg=black;option=bold>\n"
+        ."       <fg=cyan;bg=black;option=bold>Test $ psql -U #user -W template1</fg=cyan;bg=black;option=bold>"
         ."</info>"
     );
 
-    $this->config->setParameter('mysql_type', 'dsn'); // temporary limitation
-    $this->askMysqlDsn()->execute($input, $output, $dialog);
+    $this->askDbType()->execute($input, $output, $dialog);
+    $this->askDbDsn()->execute($input, $output, $dialog);
```

Comment:
We should askDbDsn() only if the type is ```mysql_dsn``` or ```pg_dsn```. For ```mysql_ram_disk```, ```mysql_mycnf```, etc, the DSN is determined automatically.